db/migrate/20200514212601_add_public_reactions_count_columns.rb

```diff
@@ -0,0 +1,8 @@
+class AddPublicReactionsCountColumns < ActiveRecord::Migration[5.2]
+  def change
+    add_column :articles, :public_reactions_count, :integer, default: 0, null: false
+    add_column :articles, :previous_public_reactions_count, :integer, default: 0, null: false
```

Comment:
Why are we adding this column?

Reply:
We have a `previous_positive_reactions_count` column that we'll want to replace too :)